index.spec.js

```diff
@@ -42,6 +42,7 @@ describe('AppAuth', () => {
     usePKCE: true,
```

Comment:
Some of the test cases seem off in this file. The `refresh` block has all the validation tests for `authorize` 🙀  - probably just a bad copy/pasta. Anyway, I'll get those shaped up.